.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Do we need this file if we're being explicit about which firebase project we're on in GitHub Actions? Or is this mostly just for fallback?

Reply:
this prob came with `firebase init`

Reply:
may be useful if we need to use firebase-cli